src/patternfly/components/List/list.scss

```diff
@@ -22,6 +40,28 @@
     list-style: var(--pf-c-list--ul--ListStyle);
   }
 
+  .pf-c-list__item {
+    display: flex;
+  }
+
+  .pf-c-list__item-icon {
+    display: flex;
+    min-width: var(--pf-c-list__item-icon--MinWidth);
+    margin-top: var(--pf-c-list__item-icon--MarginTop);
+    margin-right: var(--pf-c-list__item-icon--MarginRight);
+    font-size: var(--pf-c-list__item-icon--FontSize);
+    color: var(--pf-c-list__item-icon--Color);
+  }
+
+  &.pf-m-lg {
```

Comment:
I'm curious what you think @mcarrano @maryshak1996 about a list component with the variation `pf-m-lg` as a way to make the icons in the list larger. This would be the large list variation. I'm wondering if this should be `pf-m-large-icon` or something more specific like that, and reserve `pf-m-lg` for making a larger variation of the list itself (bigger font, more padding, etc).

Reply:
I agree @mcoker

Reply:
I'd go with, `.pf-m-icon-lg` to add a specific name and to be consistent w/`.pf-m-display-lg`.